class-wc-calypso-bridge-dotcom-features.php

```diff
@@ -34,15 +34,28 @@ function wc_calypso_bridge_has_ecommerce_features() {
 
 if ( ! function_exists( 'wc_calypso_bridge_is_ecommerce_plan' ) ) {
 	/**
-	 * Returns if a site is an eCommerce plan site or not.
+	 * Returns if a site is an eCommerce paid plan site or not.
 	 *
-	 * @return bool True if the site is an ecommerce site.
+	 * @return bool True if the site is an paid ecommerce site.
 	 */
 	function wc_calypso_bridge_is_ecommerce_plan() {
 		return WC_Calypso_Bridge_DotCom_Features::is_ecommerce_plan();
 	}
 }
 
+if ( ! function_exists( 'wc_calypso_bridge_is_wpcom_ecommerce_plan' ) ) {
+	/**
+	 * Returns if a site is an eCommerce plan site from dotCom or not.
+	 *
+	 * @since 2.1.3
+	 *
+	 * @return bool True if the site is an ecommerce from dotCom site.
```

Comment:
As above RE `dotCom` => `WordPress.com`
```suggestion
	 * Returns if a site has an eCommerce plan from WordPress.com or not.
	 *
	 * @since 2.1.3
	 *
	 * @return bool True if the site has a WordPress.com eCommerce plan.
```